src/routes/article.py

```
@@ -59,13 +57,13 @@ async def get_article_list_by_board(board_id: int, session: SessionDep, request:
 
 
 @article_general_router.get('/{id}')
-async def get_article_by_id(id: int, session: SessionDep, request: Request) -> ArticleResponse:
+async def get_article_by_id(id: int, session: SessionDep) -> ArticleResponse:
     article = session.get(Article, id)
     if not article: raise HTTPException(404, detail="Article not found")
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)